### DISCUSSION TITLE
Wallet requires full node

### DISCUSSION THREAD

**Discussion by liorwavebl** [lightningnetwork/lnd#8506] [General]
Hi,
According to the [documentation](https://github.com/lightningnetwork/lnd/blob/master/lnwallet/btcwallet/btcwallet.go#L88-L91) `btcWallet` requires a full btcd node.
Is this still correct?

**Comment by guggero** [comment]
Yeah, that comment is very old and not true anymore. Today you have a lot of options: `btcd` full node or pruned, `bitcoind` full node or pruned or Neutrino. See https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md#available-backend-operating-modes.
Feel free to create a PR to update the comment.
